dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:2.7
+RUN git clone --single-branch --branch feature_api_rest_identitat https://github.com/UPC/mailtoticket.git
```

Comment:
Crec que és mala idea clonar el repositori durant la construcció de la imatge, sobretot si el motiu és generar la imatge amb el codi d'una branca concreta.

Reply:
La meva idea era generar una imatge out-of-the-box (mínim de passos), de cara a fer les proves m'anava bé.
Per adaptar el dockerfile en un context de branca master amb fetchmail inclòs, es pot canviar el git clone per un COPY de tot.
Llavors el procés seria:
1- Git clone
2- Modificar els settings de mailtoticket i fetchmail
3- Fer el build